samples/ComputeSharp.Sample.Local/ComputeSharp.Sample.Local.csproj

```diff
@@ -7,10 +7,9 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\ComputeSharp\ComputeSharp.csproj" />
-    <ProjectReference Include="..\..\src\ComputeSharp.SourceGenerators\ComputeSharp.SourceGenerators.csproj"
-                      OutputItemType="Analyzer"
-                      ReferenceOutputAssembly="false"
-                      PrivateAssets="contentfiles;build"/>
+    <ProjectReference Include="..\..\src\ComputeSharp.SourceGenerators\ComputeSharp.SourceGenerators.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" PrivateAssets="contentfiles;build" />
```

Comment:
Can we undo this and make sure to preserve the spacing with whitespaces and not tabs?
I really don't like these super big one-lines in `.csproj` files 😄

Reply:
Yup. VS's formatter likes these for some reason.